src/Dockerfile.linux

```diff
@@ -0,0 +1,30 @@
+ARG BASE_DIR=/durabletask
```

Comment:
To get syntax coloring, suggest moving this to e.g. `src/docker/linux/Dockerfile`. https://github.com/github/linguist/issues/4566

Reply:
So I gave it a whirl and came across the issue of `ADD/COPY` requiring the source files to reside in the build context. Don't think this will work for this repo as I am adding the go source files into the image

Reply:
Yeah, can be worked around by copying the desired `Dockerfile` along with other inputs into a temp directory and running `docker build -t xxx target/xxx`, but it is more hassle.